nd4j/samediff-import/samediff-import-tensorflow/src/main/kotlin/org/nd4j/samediff/frameworkimport/tensorflow/ir/TensorflowIRGraph.kt

```diff
@@ -256,5 +257,9 @@ class TensorflowIRGraph(graphDef: GraphDef, opDef: OpList
         return false
     }
 
+    override fun convertToNDArray(tensorTypeInput: TensorProto): INDArray {
```

Comment:
@treo I added a  new utility function to graph for access during model import for dealing with ndarrays.